pkgs/tools/games/celeste/mons/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, dnfile
+, pefile
+, click
+, tqdm
+, xxhash
+, pyyaml
+, urllib3
+, platformdirs
+, update-python-libraries
+, setuptools-scm
+} :
+
+buildPythonPackage rec {
+  pname = "everest-mons";
+  version = "2.0.0";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "mons";
+    hash = "sha256-E1yBTwZ4T2C3sXoLGz0kAcvas0q8tO6Aaiz3SHrT4ZE=";
+  };
+
+  build-system = [ setuptools-scm ];
+  pyproject = true;
+  propagatedBuildInputs = [
+    dnfile
+    pefile
+    click
+    tqdm
+    xxhash
+    pyyaml
+    urllib3
+    platformdirs
+  ];
+
+  passthru.updateScript = [ update-python-libraries (toString ./.) ];
```

Comment:
```suggestion
```
We have already set `updateScript` in `buildPythonApplication`.
https://github.com/NixOS/nixpkgs/blob/e4139c0b6f2014beb41c099149f6ae447ba5bfb3/pkgs/development/interpreters/python/mk-python-derivation.nix#L222-L224